packages/db/postgres/migrations/004-add-upsert-pins-function.sql

```diff
@@ -0,0 +1,59 @@
+DROP FUNCTION IF EXISTS upsert_pin;
```

Comment:
This files is not needed anymore right? We already have a migration 004

Reply:
That's true, have removed.